src/controllers/transfers.js

```diff
@@ -61,6 +61,17 @@ function * getStateResource () {
     id.toLowerCase(), signatureType, conditionState)
 }
 
+function getPublicKey () {
+  this.body = model.getPublicKey()
```

Comment:
Is it bad practice to not use a generator if there is no code in the endpoint that will run asynchronously? The code works as written, but maybe it would be clearer to have all endpoints use a generator. I'm not sure if there are performance trade offs.